web/source/osk/bannerManager.ts

```diff
@@ -138,23 +140,27 @@ namespace com.keyman.osk {
           // Each defined option may require specialized handling.
           case 'persistentBanner':
             // Determines the banner type to activate.
-            this.alwaysShows = optionSpec['persistentBanner'];
+            this.alwaysShows = optionSpec[key];
             break;
           case 'enablePredictions':
-            // If we're not going to show suggestions, it's best to turn off predictions.
-            keyman.modelManager.enabled = optionSpec['enablePredictions'];
+            keyman.modelManager.enabledPredictions = optionSpec[key]
```

Comment:
I'm a bit squirmy about `enabledPredictions` vs `enablePredictions`... very easy to typo in the future. Can we use the same name?

Reply:
Same name, sure... but I think I might actually change the names.  Most all of the option names feel too 'wordy' / 'cumbersome' / 'technical'.  I don't mind adapting the affected code for the apps to match.

Maybe:
* `persistentBanner` -> `alwaysShow`  // Setting this _is_ done on `keyman.osk.banner`, so the semantics are currently redundant.
* `enablePredictions` -> `mayPredict`  (`mayPredict` feels semantically right as a property on `ModelManager` and makes a shorter option name, too)
* `enableCorrections` -> `mayCorrect`
* `imagePath` is fine as it is.